### PR TITLE
Events page

### PR DESCRIPTION
**Pull request by HARI-124**

### CODE REVIEW

**Comment by manu-shukla** [comment]
@Aarjav-Jain Check this PR. Event Fetching is fine now. Just a little issue with the responsiveness part

File: src/pages/EventsPage/EventsPage.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import Button from "../../components/UI/button/Button";
 const EventsPage = () => {
   const firebase = useFirebase();
   // const [eventData, seteventData] = useState([]);
-  const [eventType,setEventType] = useState("Pre-Events")
+  const [eventType,setEventType] = useState("events")
   const buttonStyle = {
```
Comment (Aarjav-Jain) [inline src/pages/EventsPage/EventsPage.jsx:13]: this is not required remove this

File: src/pages/EventsPage/EventsPage.jsx [hunk 2]
```diff
@@ -1,27 +1,74 @@
...
+import Button from "../../components/UI/button/Button";
+
 
 const EventsPage = () => {
   const firebase = useFirebase();
-  const [eventData, seteventData] = useState([]);
+  // const [eventData, seteventData] = useState([]);
+  const [eventType,setEventType] = useState("events")
+  const buttonStyle = {
+    height:"43px",
```
Comment (Aarjav-Jain) [inline src/pages/EventsPage/EventsPage.jsx:14]: this buttonStyle is not required pls remove it

Comment (HARI-124) [reply src/pages/EventsPage/EventsPage.jsx:14]: ok

**Comment by HARI-124** [comment]
@manu-shukla The button part is not responsive should I make it responsive?

**Comment by manu-shukla** [comment]
Add the greyish colour for the details section in the mobile view

**Comment by manu-shukla** [comment]
Pre-events not fetching as well

**Comment by manu-shukla** [comment]
@Aarjav-Jain Looks good to me! Merge if all good.

**Pull request merged**